.gitignore

```diff
@@ -1,5 +1,8 @@
 
-
 *.pyc
 
+# data
 data/builds/*
```

Comment:
if we are ignoring the data directory we shouldn't need to ignore anything under it (this line should be removed)

Reply:
`# data` it's just an inline comment. I am not ignoring the data folder.

Reply:
No worries at all! Will do some work on this later today :smile: